app/client/src/pages/aboutus.tsx

```diff
@@ -1,52 +1,111 @@
 import * as React from 'react';
```

Comment:
Name of this file should be in PascalCase -- `AboutUs.tsx`

Reply:
For some reason, when I try to access the page in PascalCase, it reverts to lowercase.

Reply:
Ah, my mistake.  I didn't originally notice that this was in `pages/`.  I think it should then be `kebab-case` -- `about-us`.  That would change the URL too.